jt-utilities/src/main/java/it/geosolutions/jaiext/utilities/ImageLayout2.java

```diff
@@ -0,0 +1,272 @@
+/* 
```

Comment:
If Simone is the sole author we can ask him and just use the standard jai-ext header. Check the history in jaitools.